src/zoid/buttons/util.js

```diff
@@ -77,33 +77,38 @@ export function createVenmoExperiment() : Experiment | void {
     const fundingEligibility = getFundingEligibility();
     const isEligibleForVenmo = fundingEligibility && fundingEligibility[FUNDING.VENMO] && fundingEligibility[FUNDING.VENMO].eligible;
 
-    // exclude buyers who are not eligible
-    // exclude non-desktop integrations using enable-funding=venmo
-    if (!isEligibleForVenmo || (isEnableFundingVenmo && isSupportedNativeBrowser())) {
-        return;
-    }
-
-    if (isIos() && isSafari()) {
-        return createExperiment('enable_venmo_ios', 90);
-    }
+    if (isDevice()) {
+        if (!isEligibleForVenmo || (isEnableFundingVenmo && isSupportedNativeBrowser()) || !isSupportedNativeBrowser()) {
+            return;
+        }
 
-    if (isAndroid() && isChrome()) {
-        return createExperiment('enable_venmo_android', 90);
-    }
+        if (isIos() && isSafari()) {
+            return createExperiment('enable_venmo_ios', 90);
+        }
 
-    if (!isDevice()) {
+        if (isAndroid() && isChrome()) {
+            return createExperiment('enable_venmo_android', 90);
+        }
+    } else {
         return createExperiment('enable_venmo_desktop', 5);
     }
 }
 
 export function getVenmoExperiment(experiment : ?Experiment) : VenmoExperiment {
     const enableFunding = getEnableFunding();
-    const isEnableFundingVenmo = enableFunding && enableFunding.indexOf(FUNDING.VENMO) !== -1 && isSupportedNativeBrowser();
+    const isVenmoFundingEnabled = enableFunding && enableFunding.indexOf(FUNDING.VENMO) !== -1;
+    const isNativeSupported = isSupportedNativeBrowser();
     const isExperimentEnabled = experiment && experiment.isEnabled();
 
-    return {
-        enableVenmo: Boolean(isExperimentEnabled || isEnableFundingVenmo)
-    };
+    if (isDevice()) {
+        return {
+            enableVenmo: Boolean((isExperimentEnabled || isVenmoFundingEnabled) && isNativeSupported)
+        };
+    } else {
+        return {
+            enableVenmo: Boolean(isExperimentEnabled || isVenmoFundingEnabled)
```

Comment:
I don't think `enable-funding=venmo` is suppose to show Venmo Desktop. Venmo Desktop should only be shown from being in the experiment test group.
```suggestion
            enableVenmo: Boolean(isExperimentEnabled)
```

Reply:
Thanks, @gregjopa

Reply:
@gregjopa @mnicpt would there be another way to force Venmo to show?  The `enable-funding=venmo` is convenient for testing, specially for e2e.